lib/spack/spack/installer.py

```diff
@@ -967,8 +967,20 @@ def __init__(
         self.attempts = attempts
         self._update()
 
-    def execute(self, install_status: InstallStatus) -> ExecuteResult:
-        """Execute the work of this task.
+    def start_process(self, install_status: InstallStatus):
```

Comment:
I can't change it to `start()` because it was shadowing the `start` parameter in Task. I switched it to `start_task()` instead for now, if that sounds like a reasonable name change?